api/kyverno/v1/rule_types.go

```diff
@@ -24,6 +24,8 @@ type PolicyExceptionRequestSpec struct {
 	// criteria can include resource information (e.g. kind, name, namespace, labels)
 	// and admission review request information like the name or role.
 	ExcludeResources MatchResources `json:"exclude" yaml:"exclude"`
+	// Duration defines the effective time duration of the exception.
+	Duration string `json:"duration,omitempty" yaml:"duration,omitempty"`
```

Comment:
Temporarily set `Duration` type as `string`. May need to explore how to convert it to `time.Duration`.

Also, it may not be necessarily defined in request.